src/models/converters/entries.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const _ = require('lodash')
+
+function convertToInternalEntry (data) {
```

Comment:
Yeah, they were in the model before, so I kept them around. We could delete them though.